zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -5765,6 +5765,45 @@ components:
             As a result, the completion request is rejected and the task remains active.
             Defaults to false.
           nullable: true
+        corrections:
+          $ref: "#/components/schemas/JobResultCorrections"
+    JobResultCorrections:
+      type: object
+      description: Attributes that were corrected by the worker. Others are considered not corrected.
```

Comment:
🔧 We could align the description to the [`UserTaskChangeset`](https://github.com/camunda/camunda/blob/9cb4ef558a5ed06391678a7af0a63af2cc01fb22/zeebe/gateway-protocol/src/main/proto/rest-api.yaml#L5519-L5536). This description is more explicit about the behavior: set to `null` to ignore, set to `""` to reset a string, etc. I think the biggest difference is that `assignee` is not allowed there.